plugins/modules/managed_serviceaccount_rbac.py

```diff
@@ -228,6 +264,9 @@ def ensure_managed_service_account_rbac(module: AnsibleModule, hub_client, manag
             body=new_manifest_work,
             content_type="application/merge-patch+json",
         )
+
+    #TODO detect manifestwork failure and report failure
```

Comment:
also need to add logic of detecting clusterroles/roles that are not having any clusterrolebindings/rolebindings